iOS/FVM/Framework/Resources/supported-parts.txt

```diff
@@ -14,6 +14,8 @@
 "lamp-rear-right",
```

Comment:
is is updated in scheme.json also?

Reply:
scheme.json and supported-parts.txt contain the same list of parts